src/DocumentFormat.OpenXml/Packaging/OpenXmlPackage.FlatOpc.cs

```diff
@@ -270,5 +294,79 @@ protected static Package FromFlatOpcDocumentCore(XDocument document, Package pac
             package.Flush();
             return package;
         }
+
+        /// <summary>
+        /// Represents the collection of internal package relationships.
+        /// </summary>
+        private class InternalPackageRelationships : IEnumerable<PackageRelationship>
```

Comment:
Could this be done as an iterator function instead?

Reply:
Yes, I will remove this general-purpose class and replace it with a much simpler, altChunk-specific expression.